packages/lesswrong/lib/collections/users/schema.ts

```diff
@@ -1395,7 +1395,10 @@ const schema: SchemaType<DbUser> = {
     ...notificationTypeSettingsField(),
     hidden: !dialoguesEnabled,
   },
-  
+  notificationDialogueMatch: {
+    label: "Another user and I have matched for a dialogue",
+    ...notificationTypeSettingsField({ channel: "both" }),
+  },
```

Comment:
Do we want something like "hidden: forumTypeSetting.get() === 'EAForum'" here?

Reply:
Probably `hidden: !dialoguesEnabled` (as in the one above)